pages/common/ansible-playbook.md

```diff
@@ -21,7 +21,7 @@
 
 - Run tasks in playbook for the given tags:
 
-`ansible-playbook {{playbook}} --tags {tags}}`
+`ansible-playbook {{playbook}} --tags tag1,tag2`
```

Comment:
We need the `{{}}` for placeholders but I think is good explain is a comma separated list of tags. Maybe:
```suggestion
`ansible-playbook {{playbook}} --tags {{tag1,tag2}}`
```